graph.py

```diff
@@ -107,24 +108,29 @@ def remove_edge_with_vevlb(self, vevlb):
         return self
 
     def display(self):
-        print 't # {}'.format(self.gid)
+        display_str=''
+        print('t # {}'.format(self.gid))

```

Comment:
if we add '(' and ')' when using "print", it would be better that
```Python
from __future__ import print_function
```
to print multiple strings, import print_function to prevent Py2 from interpreting it as a tuple